vtex-apps/order-placed
# [WIP] Fix/403 and invalid order

#### What is the purpose of this pull request?


Treat 403 error and invalid orders on `orderPlaced` page.

#### What problem is this solving?


When accessing an `orderPlaced` page while unlogged, the pages errors out.

Example: https://storetheme.vtex.com/checkout/orderPlaced/?og=915821966983

![image](https://user-images.githubusercontent.com/12702016/57812228-cfd68480-7742-11e9-9dd0-0520f9ab0eb0.png)

#### How should this be manually tested?

While unlogged, just check an order placed url: https://storetheme.vtex.com/checkout/orderPlaced/?og=915821966983

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/12702016/59125036-96ef9100-8937-11e9-880c-b55d4d5e1f01.png)


https://chris--storecomponents.myvtex.com/checkout/orderPlaced/?og=9158219669832

#### Types of changes
